lib/v2/yicai/dt.js

```diff
@@ -0,0 +1,107 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const types = {
+    article: 2,
+    report: 3,
+    visualization: 4,
+};
+
+module.exports = async (ctx) => {
+    const { type = 'article', category = '0' } = ctx.params;
```

Comment:
https://github.com/DIYgod/RSSHub/blob/73867c3ffcaab672f292ff95c3e2dea02bd48c3d/lib/v2/yicai/router.js#L4
Name of parameters does not match `router.js` which means `type` will always be 'article'